core/tx_pool.go

```diff
@@ -362,9 +363,12 @@ func (pool *TxPool) loop() {
 				}
 				// Any non-locals old enough should be removed
 				if time.Since(pool.beats[addr]) > pool.config.Lifetime {
-					for _, tx := range pool.queue[addr].Flatten() {
+					list := pool.queue[addr].Flatten()
+					size := len(list)
+					for _, tx := range list {
 						pool.removeTx(tx.Hash(), true)
 					}
+					queuedEvictionMeter.Mark(int64(size))
```

Comment:
Nit, please inline `int64(len(list))`. Seems weird to pre-calculate this field and store it. I was expecting something else to be done with it later.